less/index.less

```diff
@@ -57,3 +57,145 @@ html, body {
     height: 100%;
 }
 
+@link-color:        #428bca; // sea blue
+@link-color-hover:  darken(@link-color, 10%);
+@font-color: darkgray; 
+@font-type: Arial; 
+
+/* Your code starts here! */
+.container {
+	width: 900px; 
+	margin: auto;
+
+	@media (max-width: 500px) {
+		width: 100%;
+		
+	} 	
+}
+  
+header nav {
+ display: flex;
+  margin-top: 35px;
+ 
+  @media (max-width: 500px) {
+	display: flex; 
+	flex-direction: column; 
+ 
+}
```

Comment:
Good use of nested media queries here. It's worth noting that you don't need to add a second `display: flex` here since it's already added by default up above. So all you'd need to do is set the `flex-direction`.